pkg/activator/net/throttler.go

```diff
@@ -333,16 +333,33 @@ func (rt *revisionThrottler) handleUpdate(throttler *Throttler, update revisionD
 		for newDest := range update.Dests {
 			tracker, ok := trackersMap[newDest]
 			if !ok {
-				tracker = &podIPTracker{dest: newDest}
+				if rt.containerConcurrency == 0 {
+					tracker = &podIPTracker{dest: newDest, b: newInfiniteBreaker(rt.logger)}
```

Comment:
As discussed in Slack, this is wasteful. We don't need the second check as the outer breaker already told us that it's fine to move on. Same goes for the clusterIPTracker.

Would it make sense to make `podIPTracker` implement `Breaker` as well and make it specialcase on a `nil` breaker?

Reply:
Done.